.circleci/config.yml

```diff
@@ -23,14 +23,14 @@ set_npm_auth: &set_npm_auth
 restore_dependency_cache_unix: &restore_dependency_cache_unix
   restore_cache:
     keys:
-      - v8-cache-unix-{{ checksum "package.json" }}
-      - v8-cache-unix-
+      - v9-cache-unix-{{ checksum "package-lock.json" }}
```

Comment:
In order to get the tests working again we had to update our version of chromedriver. However, just updating the `packge-lock.json` file didn't work as we checksum the `package.json` file. This also means security updates that just modify the package-lock file do not cause our builds to use the latest versions.